roles/reset/tasks/main.yml

```diff
@@ -19,6 +19,7 @@
     state: absent
   with_items:
     - kubelet.service
+    - multi-user.target.wants/kubelet.service
```

Comment:
This could be done instead by adding
```
enabled: false
```
To the task above

Reply:
Yeah, y are right,  `systemctl disable` intead of delede, but for the reset operation, it should be completely cleaned up to give the user a clean environment.

Reply:
Regarding `/tmp` I think you don't need to clean it up; it should be managed by the operating system itself.

Reply:
I think it is possible to clear `/var/log/containers`, although it is all symlink.

Reply:
I think that if Kubespray generates files, Kubespray should be responsible for cleaning them up.
So the /tmp/releases and /var/log/containers also should be removed by Kubespary, what do you think? 
For multi-user.target.wants/kubelet.service also should be disabled.

Reply:
I should have made my point very clear.

`/tmp` will be cleaned up after the system reboots; we don't need to delete it.
`/var/log/containers` are all symlinks but can be cleaned up.